OpenRVS/classes/OpenUbiLogIn.uc

```diff
@@ -13,5 +13,5 @@ function ProcessGSMsg (string _szMsg)
 	HideWindow();
 	m_GameService.SaveConfig();
 	m_pSendMessageDest.SendMessage(MWM_UBI_LOGIN_SUCCESS);
-	class'OpenLogger'.static.DebugLog("SUCCESSFULLY FAKED UBI LOGIN");
+	class'OpenLogger'.static.Info("SUCCESSFULLY FAKED UBI LOGIN", self);

```

Comment:
This can be Debug() instead of Info() -- the end user doesn't need to see this and I only used this log event when I was originally writing and debugging this class.